src/components/csv-download-button.tsx

```diff
@@ -0,0 +1,59 @@
+'use client';
```

Comment:
Please rename the file to CSVDownloadButton.tsx

Reply:
Actually, all the components in this directory are named like this. Can you please verify once and if it still needs do let me know. I'll do the change.